tests/console/validate_multipath.pm

```diff
@@ -82,46 +66,57 @@ sub verify_multipath_configuration {
     record_info('configuration', 'Verify multipath configuration with multipath tool');
     my $conf_output = script_output("multipath -t");
     while (my ($k, $v) = each(%{$test_data->{attributes}})) {
-        assert_matches(qr/$k\s$v/, $conf_output, "Multipath attribute '$k $v' not found");
+        assert_matches(qr/$k\s"?$v"?/, $conf_output, "Multipath attribute '$k $v' not found");
     }
 }
 
 sub verify_multipath_topology {
-    my ($test_data) = shift;
+    my (%args) = @_;
 
-    my $wwid = $test_data->{WWID};
-    my $dm_N = $test_data->{device_map}->{hd0};
+    my $wwid      = $args{wwid};
+    my $test_data = $args{test_data};
 
     record_info('topology', 'Verify multipath topology');
-    my $topology_output = script_output("multipath -l");
-
-    # Check infomation header
-    assert_matches(qr/$wwid $dm_N/, $topology_output,
-        "Topology information does not show header info with WWID dm-N");
-    # Check paths
-    while (my ($dev, $values) = each(%{$test_data->{topology}})) {
-        # Check policy
-        assert_matches(qr/prio=$values->{prio} status=$values->{status}/, $topology_output,
-            "Policy unexpected for multipath path to $dev");
-        # Check device
-        assert_matches(qr/$dev.*active.*running/, $topology_output,
-            "Device '$dev' should be active and running");
-    }
-}
-
-sub verify_paths_list {
-    my ($test_data) = shift;
-    my $wwid = $test_data->{WWID};
-
-    record_info('paths', 'Verify multipath paths');
-    while (my ($dev, $values) = each(%{$test_data->{paths_list}})) {
-        validate_script_output("multipath -d -v3 | grep ^$wwid",
-            qr/$wwid.*$dev.*$values->{vendor_product_revision}/);
+    my $topology_output = script_output("multipath -ll");
+
+    # Check general topology info
+    my $topology    = $test_data->{topology};
+    my $ven_pro_rev = $topology->{vendor_product_revision};
```

Comment:
could you use a full words  for this variable?

Reply:
sorry, for such a compound name seems to me more proper name abbreviate with 3 letter the name of the variable.

Reply:
ignore